src/webgpu/api/validation/encoding/encoder_open_state.spec.ts

```diff
@@ -393,6 +414,104 @@ g.test('render_pass_commands')
     }, finishBeforeCommand);
   });
 
+g.test('render_bundle_commands')
+  .desc(
+    `
+    Test that functions of GPURenderBundleEncoder generate a validation error if the encoder or the
+    pass is already finished.
+
+    - TODO: Consider testing: nothing before command, end before command, end+finish before command.
```

Comment:
This TODO isn't necessary; for bundles there's only 2 cases and you're already testing both of them.
```suggestion
```

Reply:
Done.